### DISCUSSION TITLE
Godot

### DISCUSSION THREAD

**Discussion by zt64** [catppuccin/catppuccin#1859] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Yes

**Comment by boranroni** [comment]
Here are the ports I made:
https://github.com/boranroni/catppuccin-godot
I'd love to hear some feedback.

ps: I deleted my past comment on accident, my bad.

**Reply by sgoudham** [reply]
Hi there @boranroni, just a few preliminary comments!

1. Unfortunately, it seems like the repository has been generated via the template repository on the GH UI. We've since moved on from that process as the `generated from...` tag cannot be removed. Can you follow [this](https://github.com/catppuccin/catppuccin/blob/main/docs/port-creation.md#creation) process and re-create the repository please!
2. We've recently merged style guidelines for new ports, could you please ensure that the guidelines [here](https://github.com/catppuccin/catppuccin/blob/main/docs/port-creation.md#styling) are followed? (I think the main preview image and badges need to be updated accordingly!)

Hope the above is okay, thanks for your work so far on it! :heart:

**Reply by boranroni** [reply]
Hi @sgoudham , sorry it took me a while to finish it. I recreated the repo according to your feedback. 
https://github.com/boranroni/godot-engine

**Reply by vanillyn** [reply]
sorry to jump in but, the preview image is incorrect
the order is
[Latte -> Frappe -> Macchiato -> Mocha]

**Reply by boranroni** [reply]
I fixed it now, thanks for pointing it out.

**Reply by sgoudham** [reply]
Cheers! Moved it over for review in #1973
